services/app-web/src/components/Select/ProgramSelect/ProgramSelect.test.tsx

```diff
@@ -72,12 +72,43 @@ describe('ProgramSelect', () => {
         await selectEvent.openMenu(combobox)
 
         await waitFor(() => {
-            expect(screen.getByText('MSHO')).toBeInTheDocument()
-            expect(screen.getByText('SNBC')).toBeInTheDocument()
-            expect(screen.getByText('PMAP')).toBeInTheDocument()
-            expect(screen.getByText('MSC+')).toBeInTheDocument()
+            expect(screen.getAllByText('MSHO')).toHaveLength(1)
+            expect(screen.getAllByText('SNBC')).toHaveLength(2)
```

Comment:
So if rate programs are visible we would end up displaying certain programs twice even though they have the same name? That seems odd-   I'm confused by how the test has changed here and why `MSHO` has lenght 1 but `SNBC` has length 2.  Can you explain this a bit?

Reply:
I see now- there are mocked inside stateMock that set up two programs with the same nickname. In that case, please convert these tests to check for the distinct full names so we are ensuring the expected and unique program (or rate program) display